external/netstandard/netstandard.depproj

```diff
@@ -76,7 +76,17 @@
     </BinPlaceConfiguration>
   </ItemGroup>
 
-  <Target Name="AddNETStandardRefs" AfterTargets="ResolveReferences"
+  <Target Name="AddNETStandard21Refs" AfterTargets="ResolveReferences">
+    <ItemGroup>
+      <_NetStandard21Files Include="$(PackagesDir)$(NETStandardLibraryPackageId.ToLower())\$(NETStandardLibraryPackageVersion)\build\netstandard2.1\ref\*.dll" />
```

Comment:
We should error if this is empty since the wildcard will get nothing if the directory is missing.

Reply:
Maybe do the same below too, even though it wasn't introduced in this change?